include/zephyr/dt-bindings/clock/mcux_lpc_syscon_clock.h

```diff
@@ -21,6 +21,7 @@
 #define MCUX_FLEXCOMM11_CLK		11
 #define MCUX_FLEXCOMM12_CLK		12
 #define MCUX_FLEXCOMM13_CLK		13
+#define MCUX_FLEXCOMM14_CLK		14
 #define MCUX_HS_SPI_CLK			14
```

Comment:
is this right? same id for two devices? I imagine it's either shared or different soc config, just checking...

Reply:
If this is true maybe we should go with:

``
#define MCUX_FLEXCOMM14_CLK MCUX_HS_SPI_CLK
``

Reply:
they are the same

https://github.com/zephyrproject-rtos/zephyr/blob/06f9b11a2289cb4da535404b7a64670d6c66884d/drivers/clock_control/clock_control_mcux_syscon.c#L102-L107

@fabiobaltieri

> If this is true maybe we should go with:
> #define MCUX_FLEXCOMM14_CLK MCUX_HS_SPI_CLK

not worth a repush imo, would like to get this merged if no further comments

Reply:
is it? I think removing the ambiguity is worth the extra step